composer.json

```diff
@@ -58,7 +58,7 @@
     "ext-curl": "*"
   },
   "require-dev": {
-    "phpunit/phpunit": "4.5.*",
+    "phpunit/phpunit": "4.8.*",
```

Comment:
Yea, you are right, I originally made the change because I was trying to fix the flip flopping test, thinking that maybe there was a bug... and then I ended up changing that test to be skipped in all cases anyways. If this were a non "-dev" dependency, I wouldn't have bundled it together.